src/transformers/angular-fe-schema-transformer.ts

```diff
@@ -80,3 +83,23 @@ function handleSelectConceptAnswers(question: FormField) {
     };
   }
 }
+
+function handleProgramMetaTags(form: FormSchema) {
+  if (form.meta.programs.isEnrollment || form.meta.programs.discontinuationDateQuestionId) {
+    const config = {
+      programUuid: form.meta.programs.uuid
+    };
+
+    if (form.meta.programs.isEnrollment) {
+      config['enrollmentDate'] = '';
+    } else {
+      config['completionDate'] = form.meta.programs.discontinuationDateQuestionId;
```

Comment:
1. Does AFE support discontinuation from a program? If so, is it driven by `programs.discontinuationDateQuestionId`
2. What happens is there is no discontinuation date question defined in the form? Do we fallback to `new Date()`?

Reply:
Yes, the `programs.discontinuationDateQuestionId` is what is used for discontinuation, its a mandatory field.

Reply:
> Does AFE support discontinuation from a program? If so, is it driven by programs.discontinuationDateQuestionId

Reply:
@CynthiaKamau did you see the above?

Reply:
Defaulting to new date if the discontinuation date is null?

Reply:
Does the AFE support discontinuation of a program?

Reply:
How is that configured in AFE, can you link me to a documentation or an example? I'm looking for an example where we discontinue a program the AFE way

Reply:
Sample schema is [here](https://github.com/palladiumkenya/openmrs-config-kenyaemr/blob/257e9268a983c42eb291e9b516cf63ae9bc519a2/configuration/ampathforms/HIV_Discontinuation.json#L9)

AFE code lives [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/3f5b429f3d72a45c40bba18f8d49d7fb4230d32b/packages/esm-form-entry-app/src/app/openmrs-api/program-resource.service.ts#L88)

Reply:
Thanks @CynthiaKamau, clear enough!